src/Our.Umbraco.Ditto/Ditto.cs

```diff
@@ -135,5 +135,15 @@ public static void RegisterDefaultProcessorType<TProcessorAttributeType>()
         {
             TypeDescriptor.AddAttributes(typeof(TObjectType), new TypeConverterAttribute(typeof(TConverterType)));
         }
+
+        /// <summary>
+        /// Registers a global umbraco application context accessor.
+        /// </summary>
+        /// <typeparam name="TUmbracoApplicationContextAccessorType">The type of the accessor.</typeparam>
+        public static void RegisterUmbracoApplicationContextAccessor<TUmbracoApplicationContextAccessorType>()
+            where TUmbracoApplicationContextAccessorType : IUmbracoApplicationContextAccessor, new()
+        {
+            DittoProcessorRegistry.Instance.RegisterDefaultUmbracoApplicationContextAccessorType<TUmbracoApplicationContextAccessorType>();
```

Comment:
Actually, given there can be only 1 type, I'm wondering if this shouldn't just be a static variable on Ditto like ProcessorAttributeTargets and DefaultCacheBy etc. The Register methods are only really relevant if you have multiple types to register.